lib/Controller/CallController.php

```diff
@@ -158,7 +159,14 @@ public function downloadParticipantsForCall(string $format = 'csv'): DataDownloa
 
 		fseek($output, 0);
 
-		return new DataDownloadResponse(stream_get_contents($output), 'participants.csv', 'text/csv');
+		// Clean the room name
+		$cleanedRoomName = preg_replace('/-+/', '-', preg_replace('/[\/\\:*?"<>|]/', '-', $this->room->getName()));
+		// Limit to a reasonable length
+		$cleanedRoomName = substr($cleanedRoomName, 0, 100);
+		$date = (new \DateTime())->format('Y-m-d');
```

Comment:
```suggestion
		$date = $this->timeFactory->getDateTime()->format('Y-m-d');
```

Reply:
Is it in UTC time because it is coming from server?

Reply:
It would be, but we can inject the user's timezone